src/EnergyPlus/AirflowNetwork/include/AirflowNetwork/Elements.hpp

```diff
@@ -461,6 +481,72 @@ namespace AirflowNetwork {
         virtual ComponentType type() = 0;
     };
 
+    int constexpr NrInt = 20; // Number of intervals for a large opening
+
+    struct DetailedOpeningSolver
+    {
+        // Large opening variables
+        EPVector<Real64> DpProf;   // Differential pressure profile for Large Openings [Pa]
```

Comment:
More `EPVector`, less `Array1D`! Given that these are all similarly dimensioned, does it make sense to have a single `EPVector` of `struct` rather than three `EPVector` of `Real64`. Are these three quantities commonly used together?

Reply:
The detailed opening calculation didn't fare especially well in the transition to C++, so there's a lot to do on this. I wasn't sure if this solver needs to be it's own object, be part of the `Solver` object, or if the data and functions needed to be distributed to the `DetailedOpening` objects themselves. Because the detailed opening object gets used a lot, I have been reluctant to change too much, so I opted for the middle path here. I think the answer to the "are these used together" question is yes, but I'll know for sure once I dig into it more (which I am doing as part of the usability work).